tests/rptest/perf/small_batches_test.py

```diff
@@ -19,31 +19,44 @@ def omb_test(self):
         workload = {
             "name": "SmallBatchesWorkload",
             "topics": 1,
-            "partitions_per_topic": 20000,
+            "partitions_per_topic": 100,
             "subscriptions_per_topic": 1,
-            "consumer_per_subscription": 200,
-            "producers_per_topic": 200,
-            "producer_rate": 150_000,
+            "consumer_per_subscription": 1,
+            "producers_per_topic": 10,
+            "producer_rate": 30_000,
             "message_size": 1024,
             "payload_file": "payload/payload-1Kb.data",
```

Comment:
We could even use the 200 byte message size here or something

Reply:
Let me see if I can push the producer rate a bit higher with smaller messages.

Reply:
Unsurprisingly using smaller messages doesn't allow for a higher producer rate.